src/main/java/org/openlmis/buq/domain/buq/BottomUpQuantification.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org.
+ */
+
+package org.openlmis.buq.domain.buq;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import javax.persistence.PrePersist;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.javers.core.metamodel.annotation.TypeName;
+import org.openlmis.buq.domain.BaseTimestampedEntity;
+import org.openlmis.buq.dto.buq.BottomUpQuantificationLineItemDto;
+
+@Entity
+@TypeName("BottomUpQuantification")
+@Table(name = "bottom_up_quantifications", schema = "buq")
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class BottomUpQuantification extends BaseTimestampedEntity {
+
+  @NotNull
+  @Getter
+  @Setter
+  private UUID facilityId;
+
+  @NotNull
+  @Getter
+  @Setter
+  private UUID programId;
+
+  @NotNull
+  @Getter
+  @Setter
+  private UUID processingPeriodId;
+
+  @NotNull
+  @Getter
+  @Setter
+  private Integer targetYear;
+
+  @NotNull
+  @Enumerated(EnumType.STRING)
+  @Getter
+  @Setter
+  private BottomUpQuantificationStatus status;
+
+  @OneToMany(
+      mappedBy = "bottomUpQuantification",
+      cascade = {CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH, CascadeType.REMOVE},
+      fetch = FetchType.LAZY,
```

Comment:
`FetchType.LAZY` is a default setting for `OneToMany`, so it is not necessary to set it here, but I'm fine with leaving it as it is for better readability.

Reply:
I removed that line.